packages/components/src/components/contract-card/contract-card-items/__tests__/turbos-card-body.spec.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import TurbosCardBody from '../turbos-card-body';
+
+const contract_info = {
+    contract_id: 1,
+    buy_price: '1044.00',
+    profit: 50,
+    limit_order: null,
+    barrier: '10904.80',
+    current_spot_display_value: '1046.80',
+    sell_spot: '1046.80',
+    entry_spot: 1054,
+};
+
+const mockCardLabels = () => ({
+    BARRIER_LEVEL: 'Barrier level',
+    CURRENT_PRICE: 'Current price',
+    STAKE: 'Stake',
+    BUY_PRICE: 'Buy price',
+    TAKE_PROFIT: 'Take profit',
+    TOTAL_PROFIT_LOSS: 'Total profit/loss',
+    PAYOUT: 'Payout',
+    PROFIT_LOSS: 'Profit/Loss',
+    POTENTIAL_PROFIT_LOSS: 'Potential profit/loss',
+});
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(() => true),
```

Comment:
@hamza-deriv i suggest that we mock isCryptocurrency, getLimitOrderAmount, getTotalProfit, isValidToSell here instead of isMobile which is not imported/used in turbos-card-body.jsx🙂